src/lesson_05/hometask_05/Task05.java

```diff
@@ -0,0 +1,38 @@
+package lesson_05.hometask_05;
+/*Write a program where the user must enter a single word (string).
+        The program to check if the entered string is a palindrome.
+        Note: Palindrome – a word that is read left-to-right and right-to-left in the same
+        way.
+        Example:
+        Input:
+        kayak
+        wow
+        test
+        Output:
+        kayak is palindrome.
+        wow is palindrome.
+        test is not a palindrome.*/
+import java.util.Scanner;
+
+public class Task05 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please input a string:");
+        String s1 = sc.next();
+        String s2 = ""; // empty string
+        char char1;
+
+        for (int i = 0; i < s1.length(); i++) {
+            char1 = s1.charAt(i); // take 1 character from s1
+            s2 = char1 + s2; // add each char at the beginning of new string
```

Comment:
It cab be done by many ways. For example:
s1.equals(new StringBuilder(s1).reverse().toString());